homeassistant/components/tahoma/cover.py

```diff
@@ -137,14 +137,13 @@ def update(self):
         if self._closure is not None:
             if self.tahoma_device.type == HORIZONTAL_AWNING:
                 self._position = self._closure
-                self._closed = self._position == 0
             else:
                 self._position = 100 - self._closure
-                self._closed = self._position == 100
             if self._position <= 5:
                 self._position = 0
```

Comment:
Your PR description talks about this code, yet this is unchanged?

Reply:
Yes , in the PR description I talk about the fact that this is not valid to calculate the value of the closed state from the position, just before modify the position in the two following conditions (`<= 5` and `>= 95`). Before my commit, the code calculate a wrong closed state if the position is modified just after to `0` or `100`.